### PR TITLE
feat: add button component

### LINKED ISSUE #7: [front-end] Add button
To-Do:
- [x] create button (light and dark versions)

**Light version**
button-design

**Dark version**
button-design

### PR DESCRIPTION
**Pull request by eduardoPinto12**
Add basic button component, light and dark

fix #7

### CODE REVIEW

File: components/button/index.module.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@import '../../styles/variables.scss';
+
+.primary {
+	padding: 6px 12px;
+	font-size: $font-size-medium-body;
+	font-weight: bold;
```
Comment (ruisaraiva19) [inline components/button/index.module.scss:6]: Please add the `weight` variable like the `Typography` component.

File: pages/_document.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Document, { Html, Head, Main, NextScript } from 'next/document'
+
+class MyDocument extends Document {
+	render() {
+		return (
+			<Html>
+				<Head />
+				<body>
+					<script src="noflash.js" />
```
Comment (ruisaraiva19) [inline pages/_document.tsx:10]: I don't know if the file source is going to break on pages other than root.

File: components/button/index.module.scss (hunk 1 above)
Comment (eduardoPinto12) [reply components/button/index.module.scss:6]: will we have more than one font-weight for the button?

Comment (ruisaraiva19) [reply components/button/index.module.scss:6]: Maybe not, but it's good practice :)
Also, we need a transparent version of the button as well.

Comment (eduardoPinto12) [reply components/button/index.module.scss]: Added weight variable as suggested 👍

File: pages/_document.tsx (hunk 2 above)
Comment (ruisaraiva19) [reply pages/_document.tsx:10]: @eduardoPinto12 ☝️.

Comment (eduardoPinto12) [reply pages/_document.tsx:10]: I didn't fully understand this, are you suggesting to move it to the root?

Comment (ruisaraiva19) [reply pages/_document.tsx:10]: Not moving to the root, linking to the root like this: https://github.com/ruisaraiva19/favycon/pull/21/files#diff-8f3a5cf5019173ef1f56a54e45e44747R10

**Pull request merged**